packages/dev/codemods/src/s1-to-s2/src/codemods/components/Button/transform.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  commentOutProp,
+  removeProp,
+  updateComponentIfPropPresent,
+  updatePropName,
+  updatePropNameAndValue,
+  updatePropValueAndAddNewPropName
+} from '../../shared/transforms';
+import {NodePath} from '@babel/traverse';
+import * as t from '@babel/types';
+
+/**
+ * Transforms Button:
+ * - Change variant="cta" to variant="accent"
+ * - Change variant="overBackground" to variant="primary" staticColor="white"
+ * - Change style to fillStyle
+ * - Comment out isPending (it has not been implemented yet)
```

Comment:
we should fix this one https://react-spectrum.adobe.com/s2/index.html?path=/docs/button--docs not in this PR, just tasks we follow up with

Reply:
I can clean this up in a follow-up, and double check that others are updated as well.